.github/workflows/update-copyright-notices.yml

```diff
@@ -0,0 +1,39 @@
+name: Update copyright notices
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  check-and-update-copyright-notices:
+    if: ${{ github.event_name == 'push' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: ./tools/update-copyright-notices.sh
+      - run: |
+          git config --global user.email "actions@github.com"
+          git config --global user.name "github-actions[bot]"
+          pr_branch=update-copyright-notices-$(date +%Y-%m-%d-%H-%M-%S)
```

Comment:
I‘d prefer something shorter like `updateCopyrightYear/<year>`

Reply:
Thinking some more, we can even drop the year altogether. There should only ever be one active PR. If a new one is needed, the existing one can be updated (= the branch can be force-pushed). I've updated this accordingly.